decidim-core/app/views/decidim/account/_account_notification.js.erb

```diff
@@ -0,0 +1,20 @@
+var hideExistingNotification = document.querySelector("main").children[0]
+if (hideExistingNotification.classList.contains("flash") && hideExistingNotification.classList.contains("callout")) {
+  hideExistingNotification.classList.add("hide");
+}
+
+var accountNotification = document.querySelector(".account-notification");
+document.querySelector("main").prepend(accountNotification);
+
+if (accountNotification.querySelector(".account-message")) {
+  accountNotification.querySelector(".account-message").remove();
+}
+var messageDiv = document.createElement("div");
+messageDiv.classList.add("account-message")
+
+accountNotification.appendChild(messageDiv);
+messageDiv.innerHTML = "<%= @text %>";
```

Comment:
Please use JSON encoding instead because this would break in case the string contains a `"` (for one reason or another):
```suggestion
messageDiv.innerHTML = <%== @text.to_json %>;
```